docs/_includes/footer.html

```diff
@@ -1,7 +1,7 @@
 <footer class="footer">
     <div class="container">
         <div class="copyright">
-            Copyright (c) 2019, Moby Project
+            Copyright (c) 2017-2022, Moby Project

```

Comment:
We use ranges at work to more accurately portray the project lifetime.

Reply:
Thanks @Liberatys for following up on issue #55